buildscripts/push

```diff
@@ -40,9 +55,17 @@ echo "Set the build/unique image tag as: ${BUILD_TAG}"
 
 function TagAndPushImage() {
   REPO="$1"
-  TAG="$2"
+  # Trim the `v` from the TAG if it exists
+  # Example: v1.10.0 maps to 1.10.0
+  # Example: 1.10.0 maps to 1.10.0
+  # Example: v1.10.0-custom maps to 1.10.0-custom
+  TAG="${2#v}"
+
+  # Add an option to specify a custom TAG_SUFFIX
+  # via environment variable. Default is no tag.
+  # Example suffix could be "-debug" of "-dev"
+  IMAGE_URI="${REPO}:${TAG}${TAG_SUFFIX}";
```

Comment:
Where is the `TAG_SUFFIX` env? Not captured

Reply:
This means somebody set the env manually... Passing it as a function arg sounds more appropriate to me.

Reply:
It can be pushed whenever required, we are just making sure here that we can have that we are have all the tunnables to do automation.